app/src/main/java/com/duckduckgo/app/global/api/AtpPixelRemovalInterceptor.kt

```diff
@@ -54,7 +54,47 @@ class AtpPixelRemovalInterceptor @Inject constructor() : Interceptor, PixelInter
         private const val PIXEL_PREFIX = "m_atp_"
 
         // list here the pixels that except from this interceptor
-        private val PIXEL_EXCEPTIONS = listOf<String>()
+        private val PIXEL_EXCEPTIONS = listOf(
```

Comment:
This list is according to https://app.asana.com/0/1200581511062568/1204240670121966, https://app.asana.com/0/69071770703008/1204382677817470/f and https://app.asana.com/0/69071770703008/1204270498310399.